recipes/wrfplot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "wrfplot" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wrfplot-{{ version }}.tar.gz
+  sha256: 0078422303a835f68299dae893ab6ddfc63b576ac750210a43d95aea4334208c
+
+build:
+  entry_points:
+    - wrfplot=wrfplot.wrfplot:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - setuptools
+    - python >3.8
+    - setuptools-scm
+    - pip
+  run:
+    - python >3.8
+    - cartopy
+    - matplotlib-base <3.6
+    - wrf-python >=1.3
+    - imageio
+    - tqdm
+    - netcdf4
+    - basemap
```

Comment:
It is not in the metadata inquirentes. Also, it is a bit odd to require both cartopy and basemap/
```suggestion
```

Reply:
I am also aware that `basemap` is not required. But without `basemap` the recipe would not build and throw an error message `wrf-python` require `basemap` however no dependency is provided. It is really odd as `basemap`  is not mentioned as one of the dependencies in their feedstock recipe.

Reply:
Did that happened in the `pip check` test? If so you can comment it out. The problem is in `wrf-python` b/c basemap is optional but pip tries to look for it anyway.